### ISSUE TITLE
Unauthenticated access to /api/users end-point

### ISSUE THREAD

**Issue by toxa81** [statping-ng/statping-ng#280]
Hello! I noticed that in the default installations (both from dockerhub container and bare-metal using install.sh script) I can access /api/users end-point without authentication and get admin api token. Is this an expected behaviour?

**Comment by jemand771** [comment]
🤨🤨🤨

**Comment by adamboutcher** [comment]
Not intended but was the result of a fix for non admins to login.

We will look at reverting that fix.

**Comment by adamboutcher** [comment]
Related?
#266
#268
#272

https://github.com/statping-ng/statping-ng/pull/188

**Comment by NikoCat233** [comment]
Is this fixed?

**Comment by DeWaRs1206** [comment]
Wait, you're aware of some massive securities issues are present in your project, but you don't mention it anywhere ?!

**Comment by DeWaRs1206** [comment]
@NikoCat233 nope, not fixed. Be extremely careful with this issue, as the endpoint expose admin API key + credentials, allowing any attacker to easily create a new user and perform a RCE on your server via the "notification command" option, which allow to run system command from statping-ng ....
